src/main/java/com/eerussianguy/blazemap/gui/WaypointIconPreview.java

```diff
@@ -0,0 +1,41 @@
+package com.eerussianguy.blazemap.gui;
```

Comment:
wrong package, not a generic ui component.
needs to be on feature/waypoints with its friends.

Reply:
- [ ] Put `WaypointIconPreview` in more relevant package